tmpfiles.d/etc.conf.m4

```diff
@@ -15,8 +15,8 @@ t /etc/mtab - - - - security.SMACK64=_
 m4_ifdef(`ENABLE_RESOLVE',
 L! /etc/resolv.conf - - - - ../run/systemd/resolve/stub-resolv.conf
 )m4_dnl
-C /etc/nsswitch.conf - - - -
+C! /etc/nsswitch.conf - - - -
 m4_ifdef(`HAVE_PAM',
-C /etc/pam.d - - - -
+C! /etc/pam.d - - - -
 )m4_dnl
-C /etc/issue - - - -
+C! /etc/issue - - - -
```

Comment:
hmm, while we are at it, i wonder if we should switch these all to be symlinks, i.e. just replace C by L. The effect would be the same I guess, but we would benefit from vendor updates in /usr...

And the L stuff also has the factory magic built-in too...